File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        String ab = " ";
        String ba = " ";        
        String a1 = String.valueOf(a); // a 를 변환
        String b1 = String.valueOf(b); // b 를 변환
        
        ab = a1 + b1;
        ba = b1 + a1;
        
        int answer1 = Integer.parseInt(ab);
        int answer2 = Integer.parseInt(ba);
        int answer = answer1;
    
        if (answer1<answer2) {
            answer = answer2;
        }
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.js
function solution(a, b) {
    var answer = '';
    var s1 = Number(String(a)+String(b));
    var s2 = Number(String(b)+String(a));
    if(s1==s2) answer= s1;
    else {
        answer = (s1>s2)? s1:s2;
    }
    return answer;
}